azure-pipelines.yml

```diff
@@ -52,35 +60,11 @@ jobs:
       packageType: 'sdk'
       version: '2.2.x'
       performMultiLevelLookup: true
-  - task: PowerShell@2
-    displayName: "Determine suffix and populate PR title"
-    inputs:
-      targetType: 'inline'
-      script: |
-        $sourceBranch = $env:BUILD_SOURCEBRANCH
-        $buildReason = $env:BUILD_REASON
-        $title = ""
-        Write-Host "SourceBranch: $sourceBranch, Build reason: $buildReason"
-        if($sourceBranch.endsWith('master') -and ($buildReason -ne "PullRequest"))
-        {
-          "##vso[task.setvariable variable=IncludeSuffix]0" #Setting to false
-        }
-        else
-        {
-          "##vso[task.setvariable variable=IncludeSuffix]1"
-          if($buildReason -eq "PullRequest")
-          {
-            $response = Invoke-RestMethod api.github.com/repos/$env:BUILD_REPOSITORY_ID/pulls/$env:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER
-            $title = $response.title.ToLowerInvariant()
-          }
-        }
-        "##vso[task.setvariable variable=PullRequestTitle]$title"
-      pwsh: true
   - task: PowerShell@2
     displayName: "Build artifacts"
     inputs:
       filePath: '$(Build.Repository.LocalPath)\build.ps1'
-      arguments: '-buildNumber "$(buildNumber)" -includeSuffix $(IncludeSuffix) -devopsPRTitle "$(PullRequestTitle)"'
+      arguments: '-buildNumber "$(buildNumber)" -includeSuffix ([System.Convert]::ToBoolean("$(includeSuffix)")) -bypassPackaging ([System.Convert]::ToBoolean("$(bypassPackaging)")) -signOutput $false'
```

Comment:
bypassPackaging should just always be false here if we bump the condition up to the job as mentioned in my other comment.